src/components/marker.ts

```diff
@@ -17,7 +17,7 @@ import type {
 import {MapContext} from './map';
 import {arePointsEqual} from '../utils/deep-equal';
 
-export type MarkerProps = {
+export type MarkerProps = Omit<React.HTMLAttributes<HTMLElement>, 'color' | 'draggable'> & {
```

Comment:
I was not sure if `color` and `draggable` have different types that the default `HTMLElement` type so I omitted them.